src/containers/profile/TeamList.js

```diff
@@ -23,23 +23,17 @@ function ListItemLink(props) {
 }
 
 const TeamList = props => {
-  const classes = useStyles();
-  const [favorite, setFavorite] = useState({});
-  
+  const classes = useStyles();  
 
   const submit = team => {
     const teams = {
       profile_id: props.profile_id,
       team_id: team.team_id,            
       abbreviation: team.abbreviation
     }    
-
     axiosWithAuth()
       .post(`/api/favoriteTeams`, teams)
-      .then(res => {
-        //localStorage.setItem('token', res.data.payload)
-        //console.log("this token", res.data.payload);
-        console.log("res", res);
+      .then(res => {             
```

Comment:
If you aren't doing anything after a successful call, you can just leave the .then( ) out of it.